keyvault/client/secrets/README.md

```diff
@@ -0,0 +1,295 @@
+   });
+```
+
+### Delete a Secret
+
+Delete an existing Secret by calling deleteSecret.
+```Java
+SecretClient client = SecretClient.builder()
+        .endpoint("https://myvault.vault.azure.net")
+        .credentials(AzureCredential.DEFAULT)
+        .build();
+        
+secretAsyncClient.deleteSecret("secretName").subscribe(deletedSecretResponse ->
+   System.out.printf("Deleted Secret's deletion time %s \n", deletedSecretResponse.value().deletedDate().toString()));
+```
+
+## Troubleshooting
+
+### General
+
+When you interact with Azure Key Vault Secrets service using this Java client library, errors returned by the service correspond to the same HTTP status codes returned for [REST API][azkeyvault_rest] requests. For example, if you try to retrieve a Secret that doesn't exist in your Key Vault, a `404` error is returned, indicating `Not Found`.
+
+## Next steps
+Several KeyVault Java SDK samples are available to you in the SDK's GitHub repository. These samples provide example code for additional scenarios commonly encountered while working with Key Vault:
+
+### Hello World Samples
+* [HelloWorld.java](TODO) - Contains sync api scenarios found in this article.
+* [HelloWorldAsync.java](TODO) - Contains async api scenarios found in this article.
+
+### List Operations Samples
+Contains samples for following scenarios:
+* Creating Secrets
+* Listing Secrets
+* Create new version of existing secret.
+* List secret versions
+
+1. [ListOperations.java](TODO) 
+2. [ListOperationsAsync.java](TODO)
+
+### Backup And Restore Operations Samples
+Contains samples for following scenarios:
+* Create a Secret
+* Backup a Secret -- Write it to a file.
+* Delete a secret
+* Restore a secret
+
+1. [BackupAndRestoreOperations.java](TODO)
+2. [BackupAndRestoreOperationsAsync.java](TODO)
+
+
+### Managing Deleted Secrets Samples:
+Contains samples for following scenarios:
+* Create a Secret
+* Delete a secret
+* List deleted secrets
+* Recover a deleted secret
+* Purge Deleted secret
+
+1. [ManagingDeletedSecrets.java](TODO)
+2. [ManagingDeletedSecretsAsync.java](TODO) 
+    
+
+## Contributing
+This project welcomes contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the Microsoft Open Source Code of Conduct. For more information see the Code of Conduct FAQ or contact opencode@microsoft.com with any additional questions or comments.
```

Comment:
Link Microsoft Open Source Code of Conduct https://opensource.microsoft.com/codeofconduct/